admin-node-setup.sh

```diff
@@ -1,9 +1,16 @@
 #!/bin/sh
 # this script WILL BE RUN ON EVERY REBOOT
 
+# Clear the transactional update grains when booting up
+if [ -f /etc/salt/grains ]; then
+    sed -i -e 's|tx_update_reboot_needed:.*|tx_update_reboot_needed: false|g' /etc/salt/grains
+    sed -i -e 's|tx_update_failed:.*|tx_update_failed: false|g' /etc/salt/grains
```

Comment:
That's a good point. TX update will manage the lifecycle of the failed grain.

I'll remove it.